_layouts/topic.md

```diff
@@ -0,0 +1,113 @@
+    {% assign num_aliases = page.aliases | size %}
+    {% capture aliases %}{:.center}{{newline}}*Also covering&nbsp;{% endcapture %}
+    {% if num_aliases > 2 %}
+      {% for alias in page.aliases %}
+        {% if forloop.last %}
+          {% capture aliases %}{{aliases}}, and {{alias}}{% endcapture %}
+        {% elsif forloop.first %}
+          {% capture aliases %}{{aliases}}{{alias}}{% endcapture %}
+        {% else %}
+          {% capture aliases %}{{aliases}}, {{alias}}{% endcapture %}
+        {% endif %}
+      {% endfor %}
+    {% else %}
+      {% capture aliases %}{{aliases}}{{page.aliases | join: ' and '}}{% endcapture %}
+    {% endif %}
+    {% assign aliases = aliases | append: '*' %}
+  {% endif %}
+
+  <!-- Build list of primary sources -->
+  {% for source in page.primary_sources %}
+    {%- if source.link contains 'http://' or source.link contains 'https://' -%}
+      {% capture reference %}[{{source.title}}]({{source.link}}){% endcapture %}
+    {%- else -%}
+      {% capture reference %}[{{source.title}}][{{source.link}}]{% endcapture %}
+    {%- endif -%}
+    {% capture primary_sources %}{{primary_sources}}{{newline}}- {{reference}}{% endcapture %}
+  {% endfor %}
+
+  <!-- Build list of internal optech mentions -->
+  {% assign references = '' %}
+  {% assign mentions = page.optech_mentions | sort: "date" | reverse %}
+  {% for mention in mentions %}
+    {% if mention.feature == true %}
+      {% assign bold = site.bold %}
+    {% else %}
+      {% assign bold='' %}
+    {% endif %}
+    {% capture references %}{{references}}{{newline}}- {{bold}}{{mention.date}} [{{mention.title}}]({{mention.url}}){{bold}}{% endcapture %}
+  {% endfor %}
+
+  <!-- Build list of see also entries -->
+  {% for source in page.see_also %}
+    {%- if source.link contains 'http://' or source.link contains 'https://' -%}
+      {% capture reference %}[{{source.title}}]({{source.link}}){% endcapture %}
+    {%- else -%}
+      {% capture reference %}[{{source.title}}][{{source.link}}]{% endcapture %}
+    {%- endif -%}
+    {% capture see_also %}{{see_also}}{{newline}}- {{reference}}{% endcapture %}
+  {% endfor %}
+
+  <!-- Variable for use in links -->
+  {% capture gh_base %}https://github.com/{{site.github_username}}/{{site.repository_name}}{% endcapture %}
+{% endcapture %}
+
+<!-- Actual page content -->
+{% capture content %}
+  {% include references.md %}
+  {{aliases}}
+
+  {{page.excerpt}}
+
+  {{page.extended_summary}}
+
+  {%- if page.primary_sources and page.primary_sources != '' -%}
+    ## Primary code and documentation
+
+    {{primary_sources}}
+  {% endif %}
+
+  {%- if page.optech_mentions and page.optech_mentions != '' -%}
+    ## Optech newsletter and website mentions
+
+    {{references}}
+  {% endif %}
+
+  {%- if page.see_also and page.see_also != '' -%}
+    ## See also
+
+    {{see_also}}
+  {% endif %}
+
+  <br>
+  <span class="float-left" markdown="1">**Previous:**<br>{{prev_link}}</span>
```

Comment:
Good point.  I could change the text to say "Previous main topic" and "Next main topic" which might make it less confusing at the cost of extra verbosity.

Reply:
Still feels unclear to me. I dont have a better suggestion though. Other than omitting the prev/next links altogether.

@jnewbery opinion here?

Reply:
I think the only reason people would use these is if they were browsing around casually, so I don't think it's that important that the ordering matches the index page that they originated from. At the risk of bike-shedding, I'd suggest a couple of small changes:

- Label these as **Previous Topic** / **Next Topic** instead of **Previous** / **Next**
- In the `en/topics` page, for any aliases, list them as `{{alias}} (see [{{topic.name}}]({{topic.url}}))` so it's obvious that the user is being taken to the `topic` page, not `alias` page.

I've implemented those suggestions here: https://github.com/jnewbery/bitcoinops.github.io/commit/b6d0bddc111fbd1007c25ffecfb8e94b3697f629 . Let me know what you think.

Reply:
I think that makes the list significantly harder to read because the eye (at least my eye) tends to follow the same text color rather than reading vertically.

![2019-10-18-06_11_58_368095233](https://user-images.githubusercontent.com/61096/67110344-3a667300-f16e-11e9-876a-2cf8ac2a27c4.png)

I think the goal here is just to communicate that some list items point to the same topic entries as other list items, so I italicized the aliases (which is a bit hard to see, but it's possible if you care enough) and added a note to the top of the page:

![2019-10-18-06_12_44_376877702](https://user-images.githubusercontent.com/61096/67110460-82859580-f16e-11e9-9e48-0003ea101db8.png)

Of course, I also kept the addition of "Topic" to the Previous/Next links.

Reply:
That's better but I still think it makes it harder for people to quickly find the entry they want.  Also, with the whole link linked, it's less clear what they're clicking on (e.g. are they supposed to click that link or are they supposed to scroll from BIP156 to Dandelion and click the Dandelion link?)  

For desktop users, I can easily add a `title` attribute to the link so hovering over a link will tell them what it points to (this is already possible by just looking where the link points).

Reply:
I wouldn't bother with a `title` attribute. You've mentioned before that we shouldn't rely on that as part of the UX, since it's difficult for mobile/tablet users to access, which I agree with.

Reply:
RE original thread:

> Label these as Previous Topic / Next Topic instead of Previous / Next

ACK

RE listing aliases on topic list pages: I dont have a strong preference for any of the 3 options discussed above. As is (italics for aliases), works for me personally.